enacrestic/restic_backup.py

```diff
@@ -208,9 +208,7 @@ def _handle_stderr(self):
                     ResticCompletionStatus.REPO_NOT_INITIALIZED
                 )
 
-            elif re.search(
-                r"the `unlock` command can be used to remove stale locks", stderr
-            ):
+            elif re.search(r"Fatal: unable to create lock in backend", stderr):
```

Comment:
Should you use only `r"unable to create lock in backend"`? Because `Fatal: ` does not seem to appear with restic 0.12.1 according to the PR description.

Reply:
🤪 You're right ! thanks for the review : )